libdevcore/Log.cpp

```diff
@@ -43,7 +43,7 @@ void dev::simpleDebugOut(std::string const& _s, char const*)
 	static Mutex s_lock;
 	Guard l(s_lock);
 
-	cerr << _s << endl << flush;
+	cout << _s << endl << flush;
```

Comment:
in the case of eth, there is "expected, domain-specific output" (stdout) and "out of band output" (stderr); logs are out of band and include debug info, notes, warnings and critical errors. 

it's a convention that i employ and it isn't going to change. if you want to employ different meanings for neth, please do.